cc/main.cc

```diff
@@ -87,6 +87,8 @@ DEFINE_bool(soft_pick, true,
 DEFINE_bool(random_symmetry, true,
             "If true, randomly flip & rotate the board features before running "
             "the model and apply the inverse transform to the results.");
+DEFINE_string(init_strategy, "parent",
```

Comment:
Do we need init to draw?  Would it be better to make this a float from [0,-2] (with clamping, naturally)?